lib/release.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const newGithubReleaseUrl = require('new-github-release-url');
+const opn = require('opn');
+
+module.exports = options => {
+	const url = newGithubReleaseUrl({
+		repoUrl: options.repoUrl,
+		tag: options.version,
```

Comment:
The `tag` here needs to adhere to the tagPrefix. See the logic in https://github.com/sindresorhus/np/blob/d5ef58ba699a3687e9a89ca72ca23965f496cf1c/lib/prerequisite.js#L96-L107 (Probably smart to export that logic to an [utility function](https://github.com/sindresorhus/np/blob/master/lib/util.js)). Right now, I get `0.23.30`, while my project (and most npm packages) uses `v0.23.30` type tags.